mitiq/zne/scaling/folding.py

```diff
@@ -39,13 +40,28 @@ class UnfoldableCircuitError(Exception):
     ops.Z: "Z",
     ops.T: "T",
     ops.I: "I",
+    ops.S: "S",
+    ops.T: "T",
+    ops.rx: "RX",
+    ops.ry: "RY",
+    ops.rz: "RZ",
```

Comment:
```suggestion
    ops.rx: "rx",
    ops.ry: "ry",
    ops.rz: "rz",
```
For consistency with docstring? I know this is not so important since you convert to lowercase anyway.